docs/commands/GetCurrentMC.md

```diff
@@ -64,39 +64,39 @@ It's pulse counter's values for selected channels.
 
 #### 4 first channels:
 
-| Field            | Value | Bits                                                                    | Hex      |
-| ---------------- | ----- | ----------------------------------------------------------------------- | -------- |
-| command id       | `24`  |                                                                         | `0x18`   |
-| command size     | `6`   |                                                                         | `0x06`   |
-| channels         |       | `0b00001111`                                                            | `0x0f`   |
-| channel #0 value | `131` | `0b0000000010000011` <br> with extended bits: <br> `0b0000000110000011` | `0x8301` |
-| channel #1 value | `8`   |                                                                         | `08`     |
-| channel #2 value | `10`  |                                                                         | `0a`     |
-| channel #3 value | `12`  |                                                                         | `0c`     |
+| Field             | Value              | Bits                                                                    | Hex      |
+| ----------------- | ------------------ | ----------------------------------------------------------------------- | -------- |
+| command id        | `24`               |                                                                         | `0x18`   |
+| command size      | `6`                |                                                                         | `0x06`   |
+| channels          | `1`, `2`, `3`, `4` | `0b00001111`                                                            | `0x0f`   |
+| channel `1` value | `131`              | `0b0000000010000011` <br> with extended bits: <br> `0b0000000110000011` | `0x8301` |
```

Comment:
does it good to make an example with real INT bit to explain EXTENDED value? Will be good example at that special type, here - these integer values don't exists in real

Reply:
why they don't exist? `131` is very real value of the first channel